src/SearchResults.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import moment from "moment";
+
+const headings = [
+  "Id",
+  "Title",
+  "First Name",
+  "Surname",
+  "Email",
+  "room Id",
+  "Check-In",
+  "Check-Out",
+  "Night"
+];
+const SearchResults = props => {
+  return (
+    <table className="table">
+      <thead>
+        <tr>
+          {headings.map(heading => {
+            return <th scope="col">{heading}</th>;
```

Comment:
We are returning `jsx` in a map here. We need to give the first element a `key` prop. Look at other comment for details, or in your browser console.

The correct code is
```
<thead>
	<tr>
		{headings.map((heading, index) => {
			return <th key={index} scope="col">{heading}</th>;
		})}
	</tr>
</thead>
```

Reply:
Done!